tools/unix/build_omim.sh

```diff
@@ -146,8 +145,10 @@ build()
       done
     fi
   else
-    "$CMAKE" "$CMAKE_GENERATOR" "$OMIM_PATH" -DCMAKE_BUILD_TYPE="$CONF" \
-    -DBUILD_DESIGNER:bool=True ${CMAKE_CONFIG:-}
+    "$CMAKE" "$CMAKE_GENERATOR" "$OMIM_PATH" \
+      -DCMAKE_BUILD_TYPE="$CONF" \
```

Comment:
1. Could it be a one-liner?
2. 4 spaces indent for the same command is clearer.

Reply:
1. I suppose it could, but it felt quite a long line (>100 chars long). The other comparably long line is one of the help ones.
2. Noted!

Reply:
We're ok with 120 chars per line. It fits nicely into all modern monitors and the Github review window.

Reply:
Ah alright, will collapse into one line then.